notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #90.            dataflow_use_public_ips=dataflow_use_public_ips,
</pre><p>Do we want to shows users these fields in this notebook? From lines 85 to 90. They are more advanced functionality &amp; I don't see explanations around these fields. Maybe a callout to the docs page about additional dataflow configuration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>